packages/core-flows/src/definition/cart/steps/get-variant-price-sets.ts

```diff
@@ -0,0 +1,82 @@
+import { ModuleRegistrationName } from "@medusajs/modules-sdk"
```

Comment:
A general question - Should getters be a step in the workflow, or should they just be helper functions? Since we won't have a compensation function for getters, and usually they are just as a support role, performance will likely be worse if they are a step in the workflow.